includes/admin/meta-boxes/views/html-order-download-permission.php

```diff
@@ -5,30 +5,48 @@
 ?>
 <div class="wc-metabox closed">
 	<h3 class="fixed">
-		<button type="button" data-permission_id="<?php echo esc_attr( $download->get_id() ); ?>" rel="<?php echo esc_attr( $download->get_product_id() ) . ',' . esc_attr( $download->get_download_id() ); ?>" class="revoke_access button"><?php _e( 'Revoke access', 'woocommerce' ); ?></button>
+		<button type="button" data-permission_id="<?php echo esc_attr( $download->get_id() ); ?>" rel="<?php echo esc_attr( $download->get_product_id() ) . ',' . esc_attr( $download->get_download_id() ); ?>" class="revoke_access button"><?php esc_html_e( 'Revoke access', 'woocommerce' ); ?></button>
 		<div class="handlediv" aria-label="<?php esc_attr_e( 'Click to toggle', 'woocommerce' ); ?>"></div>
-		<strong><?php
+		<strong>
+			<?php
 			printf(
 				'#%s &mdash; %s &mdash; %s: %s &mdash; ',
 				esc_html( $product->get_id() ),
-				esc_html( apply_filters( 'woocommerce_admin_download_permissions_title', $product->get_name(), $download->get_product_id(), $download->get_order_id(), $download->get_order_key(), $download->get_download_id() ) ),
+				esc_html(
+					apply_filters(
+						'woocommerce_admin_download_permissions_title',
+						$product->get_name(),
+						$download->get_product_id(),
+						$download->get_order_id(),
+						$download->get_order_key(),
+						$download->get_download_id()
+					)
+				),
 				esc_html( $file_count ),
-				esc_html( wc_get_filename_from_url( $product->get_file_download_path( $download->get_download_id() ) ) )
+				esc_html(
```

Comment:
I don't think this is necessary? Same for the above filter. Unless there is a PHPCS issue, it should be left.

Reply:
Yes, it was detected by **PHPCS**.

Reply:
Not using our `phpcs.ruleset.xml`.

Reply:
Not on my install...these types of change are unnecessary. cc @claudiosanches

Reply:
I don't know which standard you are using, but I'm using *WordPress VIP Coding Standard*. @mikejolley and @claudiosanches

Reply:
I removed the escaping from both this and the above filter. @mikejolley